RedHatInsights/insights-inventory-frontend
# Refactor the /routes/InventoryTable into hybridInventory tabs

This PR is intended to consolidate the current state of /routes/InventoryTable component by extracting the tabs into the hybridInventoryTabs module and preparing the insights-inventory to expose this new module by federation.

How to test:
1. Run the PR locally
2. No change should be present in functionality at this phase
